modules/google-maps/src/google-maps-overlay.js

```diff
@@ -166,7 +166,9 @@ export default class GoogleMapsOverlay {
       // the GL state can be inconsistent
       setParameters(gl, {
         scissor: [0, 0, gl.canvas.width, gl.canvas.height],
-        stencilFunc: [gl.ALWAYS, 0, 255, gl.ALWAYS, 0, 255]
+        stencilFunc: [gl.ALWAYS, 0, 255, gl.ALWAYS, 0, 255],
+        depthMask: true,
```

Comment:
This does not seem right. Picking does not alter `depthMask` and `depthTest`. The places we call `setParamters` instead of `withParameters` are:

- On WebGL initialization with defaults
- Before each redraw with `deck.props.parameters`
- Before each pass with `viewport` and `framebuffer`
- For each layer with `polygonOffset`

Reply:
I've tested it more and you are right. The issue is not resolved by this fix, but instead occurs seemingly randomly. I think the regression is down to a change in Google's library (which is updated weekly) as I did not see this when merging&testing the Google PR a few weeks back

Reply:
We are working on this, but no clear fix yet. Instead I've added a few other bug fixes so we can merge this PR